Formula/qt.rb

```diff
@@ -129,31 +129,30 @@ class Qt < Formula
   def install
     # Install python dependencies for QtWebEngine
     venv = virtualenv_create(buildpath/"venv", "python3")
-    resources.each do |r|
-      venv.pip_install r
-    end
-    xy = Language::Python.major_minor_version "python3"
-    ENV.prepend_path "PYTHONPATH", buildpath/"venv/lib/python#{xy}/site-packages"
-    ENV.prepend_path "PATH", Formula["python@3.10"].libexec/"bin"
+    venv.pip_install resources
+
+    ENV.prepend_path "PYTHONPATH", buildpath/"venv"/Language::Python.site_packages("python3")
+    ENV.prepend_path "PATH", Formula["python@3.10"].opt_libexec/"bin"
```

Comment:
Maybe set `PATH` before other `"python3`" usage to help in next Python 3.11 migration?

Reply:
I guess doesn't really matter if https://github.com/Homebrew/brew/pull/13677 is merged